src/rabbit_control_main.erl

```diff
@@ -547,6 +549,16 @@ action(eval, Node, [Expr], _Opts, _Inform) ->
 action(help, _Node, _Args, _Opts, _Inform) ->
     io:format("~s", [rabbit_ctl_usage:usage()]);
 
+action(node_health_check, Node, _Args, _Opts, Inform) ->
+    Inform("Health check of node ~p", [Node]),
+    try
+        rabbit_health_check:node(Node),
+        io:format("Node ~p is up and running~n", [Node])
+    catch
+        {node_is_ko, ErrorMsg} ->
+            io:format("~s~nProblems encountered in node ~p~n", [ErrorMsg, Node])
```

Comment:
Currently there is only one possible reason - `node_is_ko`. EX_SOFTWARE (70) suits it the best.

And is it possible to add separate reason with exit code EX_NOHOST (68) for cases when a node is unavailable?